dashboard/app/models/concerns/pd/jot_form_backed_form.rb

```diff
@@ -169,10 +177,29 @@ def _sync_from_jotform(form_ids)
         imported
       end
 
-      def process_submission(submission)
+      # Process a submission and saves the model.
+      # @param [submission] (optional) The submission received from JotForm.
```

Comment:
Is `submission` actually optional?

Reply:
Will address in https://github.com/code-dot-org/code-dot-org/pull/28850.